ephemeraldata.adoc

```diff
@@ -35,40 +35,85 @@ through the following process:
-backend, the value of `zimbraEphemeralBackendURL` will not be changed.
+[start=2]
+. Migrate existing ephemeral data to the `SSDB` backend using the `zmmigrateattrs` utility
 
 ----
 sudo su - zimbra
-zmprov mcf zimbraEphemeralBackendURL ssdb:<ip address|hostname>:port # substituting your server values
+/opt/zimbra/bin/zmmigrateattrs ssdb:<ip address|hostname>:port # substituting your server values
 ----
 
+You may use either an ip address or a hostname for the host portion of the destination URL.
+Either way, you will need to ensure it resolves to the proper ip address on all machines in the cluster.
+If the provided SSDB address do not resolve to a functioning backend, the migration process will terminate.
+
+
 [start=3]
-. Migrate any existing attributes from LDAP -> SSDB
+. Configure {product-name} to use `SSDB`:
 
 ----
 sudo su - zimbra
-/opt/zimbra/bin/zmmigrateattrs -n 4
+zmprov mcf zimbraEphemeralBackendURL ssdb:<ip address|hostname>:port # substituting your server values
 ----
 
-== Additional Notes on Ephemeral Data
+As with migration, the host and port must resolve to a functioning SSDB backend. Otherwise,
+the value of `zimbraEphemeralBackendURL` will not be changed.
+
+
+== Migration Details
+
+=== Migration Info
+
+Information about the latest migration process can be viewed by running the command `zmmigrateattrs --status`.
+If the migration is currently in progress, this command may have to be run from a new terminal window.
+This command will output three pieces of information:
+
+1. The status of the migration: one of IN_PROGRESS, COMPLETED, or FAILED
+2. The URL of the SSDB backend acting as the destination
+3. A timestamp of when the migration process was initiated
+
+The migration info can be reset with the command `zmmigrateattrs --clear`. This should only be done if
+the status does not reflect the true state of the system.
+
+=== Changing the Ephemeral Backend URL
+
+When the value of zimbraEphemeralBackendURL is modified, the {product-name} checks the status of the last known migration.
+This can result in one of several scenarios:
+
+1. If the migration is completed and the URL of this migration matches the newly provided value, zimbraEphemeralBackendURL
+is changed to the new value and the migration info is reset. This is the expected use case.
+2. If a migration is currently in progress, zimbraEphemeralBackendURL will not be changed.
+3. If no migration info is available, if the migration has failed, or if the new URL does not match the migration URL,
+zimbraEphemeralBackendURL will be changed; however, a warning will be logged stating that data is not guaranteed to have
+been migrated.
+
+
+=== Forwarding Ephemeral Data
+
+During the migration process, and until the backend URL is changed, {product-name} will store new ephemeral data
+both in LDAP and `SSDB`; this keeps the two backends getting out of sync. If the new value of zimbraEphemeralBackendURL
```

Comment:
I didn't emphasize it because it's not emphasized in other parts of the doc either. Not sure what the most consistent approach would be.